### PR TITLE
Changes for connecting to cloud mongodb server

### PR DESCRIPTION
**Pull request by dangunter**
## Fixes/Addresses:

You can connect to cloud instances of MongoDB by just changing the URL, but one problem that can occur is a failure to verify certificates (over SSL), so this adds some code to retry using the `certifi.where()` module, if the certifi package is installed.

In the process of testing and documentation, some Jupyter Notebooks were added to the repo. These are kept in sync by running them automatically.

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: watertap/edb/db_api.py [hunk 1]
```diff
@@ -265,6 +323,9 @@ def get_base(self, name: str = None) -> Union[Result, Base]:
         else:
             return result
 
+    # older method name
+    get_one_base = get_base
+
```
Comment (aladshaw3) [inline watertap/edb/db_api.py:341]: Do we plan on phasing this older method out? If so, should we have some kind of depreciation warning (does that even matter of us at this point) ?

File: watertap/tests/test_notebooks.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+# =====
+
+
+@pytest.mark.component
+def test_edb_notebooks(docs_root, electrolytedb):
+    for nb_path in find_notebooks(docs_root / "examples" / "edb"):
+        if ".ipynb_checkpoints" in nb_path.parts:
+            continue
+        print(f"run notebook: {nb_path}")
+        run_notebook(nb_path)
```
Comment (aladshaw3) [inline watertap/tests/test_notebooks.py:76]: Quick question: This is the directory ('docs/examples/edb') in which I should put my other edb examples I plan on making, correct? And then this code should automatically check those, right?

**Review by aladshaw3 — APPROVED** [review]
LGTM. Just a few brief questions in my review.

File: watertap/edb/db_api.py (hunk 1 above)
Comment (aladshaw3) [reply watertap/edb/db_api.py:341]: This function is now unneeded

File: watertap/tests/test_notebooks.py (hunk 2 above)
Comment (aladshaw3) [reply watertap/tests/test_notebooks.py:76]: yes

**Review by adam-a-a — APPROVED** [review]
LGTM

**Pull request merged**